packages/react-components/src/organisms/SpeakersList.tsx

```diff
@@ -0,0 +1,156 @@
+import { tabletScreen, perRem, mobileScreen } from '../pixels';
+import { userPlaceholderIcon } from '../icons';
+import { useDateHasPassed } from '../date';
+
+const getPlaceholderAvatarUrl = () =>
+  `data:image/svg+xml;base64,${btoa(
+    renderToStaticMarkup(userPlaceholderIcon),
+  )}`;
+
+const gridStyles = css({
+  display: 'grid',
+  flexFlow: 'column',
+  gap: `${15 / perRem}em`,
+  [`@media (max-width: ${mobileScreen.width}px)`]: {
+    gap: 0,
+  },
+});
+
+const userStyles = css({
+  overflow: 'hidden',
+  display: 'grid',
+  gridTemplateColumns: 'min-content 1fr',
+  gridColumnGap: `${6 / perRem}em`,
+  alignItems: 'center',
+});
+
+const gridMixin = {
+  display: 'grid',
+  [`@media (min-width: ${tabletScreen.width}px)`]: {
+    gridTemplateColumns: '1fr 1fr 1fr',
+    gridAutoFlow: 'column',
+    alignItems: 'start',
+  },
+};
+
+const speakerListStyles = css({
+  ...gridMixin,
+  [`@media (max-width: ${mobileScreen.width}px)`]: {
+    gridAutoFlow: 'row',
+    alignItems: 'start',
+    borderBottom: '1px solid #DFE5EA',
+    [`&:nth-of-type(n+4)`]: {
+      display: 'none',
+    },
+  },
+  [`&:nth-of-type(n+6)`]: {
+    display: 'none',
+  },
+});
+
+const headerStyle = css({
+  ...gridMixin,
+  [`@media (max-width: ${mobileScreen.width}px)`]: {
+    display: 'none',
+  },
+});
+
+const labelStyle = css({
+  [`@media (max-width: ${mobileScreen.width}px)`]: {
+    gridTemplateColumns: '1fr 1fr 1fr',
+    gridAutoFlow: 'column',
+    alignItems: 'start',
+    display: 'inline-block',
+    fontWeight: 'bold',
+  },
+  display: 'none',
+});
+
+const groupStyle = css({
+  display: 'flex',
+  flexFlow: 'column',
+  [`@media (max-width: ${mobileScreen.width}px)`]: {
+    paddingBottom: `${20 / perRem}em`,
+    paddingTop: `${20 / perRem}em`,
+  },
+});
+
+interface SpeakerListProps {
+  speakers: EventSpeaker[];
+  readonly endDate: string;
+}
+
+const SpeakerList: React.FC<SpeakerListProps> = ({ speakers, endDate }) => {
+  const considerEndedAfter = addHours(
+    parseISO(endDate),
+    EVENT_CONSIDERED_PAST_HOURS_AFTER_EVENT,
+  );
+
+  const hasEnded = useDateHasPassed(considerEndedAfter);
+  const userToBeAnnounced = hasEnded
+    ? 'User was never announced'
+    : 'To be announced';
+
+  return (
+    <div>
+      <Headline3 styleAsHeading={3}>Speakers</Headline3>
+      <div css={headerStyle}>
+        <Headline4 styleAsHeading={4}>Team</Headline4>
+        <Headline4 styleAsHeading={4}>Speaker</Headline4>
+        <Headline4 styleAsHeading={4}>Role</Headline4>
+      </div>
+      <div css={gridStyles}>
+        {speakers.map(({ user, team, role }) => (
+          <div key={team.id + user?.id} css={speakerListStyles}>
+            <div css={groupStyle}>
+              <div css={labelStyle}>
+                <span>Team</span>
+              </div>
+              <Link href={network({}).teams({}).team({ teamId: team.id }).$}>
+                {team?.displayName}
+              </Link>
```

Comment:
check link has the correct displayName and Id.

Here team has a ? at the end, I don't think this is necessary